packages/controllers/src/multichain/MultiChainController.ts

```diff
@@ -301,6 +346,16 @@ export class MultiChainController extends BaseController<
     });
   }
 
+  /**
+   * Send a request to the given Snap.
```

Comment:
Not sure if we want to clarify, but this specifically calls `keyring[method](...params)` for a given snap id.